setup.py

```diff
@@ -748,4 +748,5 @@ def pxd(name):
       long_description=LONG_DESCRIPTION,
       classifiers=CLASSIFIERS,
       platforms='any',
+      python_requires='>2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*',
```

Comment:
hmm, there has to be a better way to write this, so signify *only* 2.7 or >=3.5

Reply:
I was following numpy (https://github.com/numpy/numpy/blob/master/setup.py#L368) and the link I mentioned.

Reply:
https://packaging.python.org/tutorials/distributing-packages/?highlight=python_requires#python-requires

Reply:
I agree that it should be simpler but wasn't sure what conditionals were allowed.

Reply:
Oh, I do notice a big mistake. It should be `>=2.7` not `>2.7`. Researching to see if there is a shorter way.

Reply:
```
In [15]: from pip.utils.packaging import specifiers, version

In [16]: version.parse('3.5.0') in specifiers.SpecifierSet('==2.7,>=3.5.*')
Out[16]: False
```

Reply:
I *think* you can do ``>=3.5`` ?